phpBB/phpbb/cache/driver/apcu.php

```diff
@@ -25,7 +25,10 @@ class apcu extends \phpbb\cache\driver\memory
 	*/
 	function purge()
 	{
-		apcu_clear_cache();
+	/* use an iterator to selectively clear our cache entries without
```

Comment:
Indentation seems to be off and comment style does not fit with our other code. E.g. this would work:
```
		/*
		* Use an iterator to selectively clear our cache entries without
		* disturbing any other cache users (e.g. other phpBB boards hosted on this server)
		*/
```

Reply:
Ok.  I accept that the comment format is not acceptable.
I am away from my desk machine for 2 weeks, but can delete the comment on GitHub if you prefer.